README.md

```diff
@@ -101,6 +101,9 @@ At the first startup, LibrePCB asks for a workspace directory where the library
 elements and projects will be saved.  For developers there is a demo workspace
 inclusive library and projects in the submodule "dev/demo-workspace/".
 
+*Note: As LibrePCB is still in developement, it's highly recommended to use this 
+demo workspace. Otherwise you won't be able to create/use any components and some 
+parts of the application will not work properly.*
```

Comment:
I would put a bigger note right after the installation. Here people will overlook it.